ria_remote/create_sibling_ria.py

```diff
@@ -0,0 +1,305 @@
+    ids. The thereby created dataset directories contain a bare git repository.
+    Those bare repositories are slightly different from plain git-annex bare repositories in that they use the standard
+    dirhashmixed layout beneath annex/objects as opposed to dirhashlower, which is git-annex's default for bare
+    repositories. Furthermore, there is an additional directory 'archives' within the dataset directories, which may or
+    may not contain archives with annexed content.
+    Note, that this helps to reduce the number of inodes consumed (no checkout + potential archive) as well as it allows
+    to resolve dependencies (that is (sub)datasets) merely by their id.
+    Finally, there is a file `ria-layout-version` put beneath the store's base path, determining the version of the
+    dataset tree layout and a file of the same name per each dataset directory determining object tree layout version
+    (we already switch from dirhashlower to dirhashmixed for example) and an additional directory `error_logs` at the
+    toplevel.
+    """
+
+    # TODO: option to skip existing remotes in case of recursive?
+
+    _params_ = dict(
+        dataset=Parameter(
+            args=("-d", "--dataset"),
+            doc="""specify the dataset to process.  If
+            no dataset is given, an attempt is made to identify the dataset
+            based on the current working directory""",
+            constraints=EnsureDataset() | EnsureNone()),
+        sibling=Parameter(
+            args=("sibling",),
+            metavar="SIBLING",
+            doc="""name of the to be created sibling""",
+            constraints=EnsureStr() | EnsureNone()),
+        storage_sibling=Parameter(
+            args=("-s", "--storage-sibling"),
+            metavar="STORAGE",
+            doc="""name of the RIA storage sibling. Must not be identical to SIBLING. By default SIBLING is appended 
+            with '-storage'""",
+            constraints=EnsureStr() | EnsureNone()),
+        force=Parameter(
+            args=("-f", "--force"),
+            doc="""don't fail on existing siblings. Use and possibly reconfigure them instead.""",
+            action='store_true'),
+        no_publish=Parameter(
+            args=("--no-publish",),
+            doc="""whether to publish the dataset's history (no data) to SIBLING after creation.""",
+            action="store_true"),
+        no_server=Parameter(
+            args=("--no-server",),
+            doc="""don't assume the store to be served by a webserver. By default git's update-server-info hook is 
+            enabled. With this option that can be disabled.""",
+            action="store_true"),
+        recursive=recursion_flag,
+    )
+
+    @staticmethod
+    @datasetmethod(name='create_sibling_ria')
+    @eval_results
+    def __call__(
+            sibling,
+            dataset=None,
+            storage_sibling=None,
```

Comment:
I am unsure, if this argument setup is optimal/needed. I am still thinking that a `ria+ssh://example.com/storepath`-like URL, pointing to the target store should be all that is needed.